app/models/course_session.rb

```diff
@@ -20,10 +22,25 @@ def accepting_applications
   # Instance methods
   #
   def start_to_s
-    return self.start_date.strftime('%m-%d-%y')
+    return self.start_date.strftime('%m-%d-%y') if self.start_date
   end
 
   def destroy_lectures
     self.lectures.destroy_all
   end
+
+  def status_badge
```

Comment:
I would probably do this with a lookup table, e.g.

status_badges = {
  draft: "light",
  admissions: "primary",
  ...
}

class_tag = "badge badge-pill badge-#{status_badges[status]}"

Reply:
Updated so that this uses a lookup table. Took me a little to understand that they key in this instance is a symbol, but works now.